swan-lake/development-tutorials/source-code-dependencies/organize-ballerina-code.md

```diff
@@ -14,7 +14,7 @@ Writing code in an organized manner from the beginning of the project is importa
 
 Ballerina code is organized in a single shareable unit called a `package`.
```

Comment:
Not introduced by this PR, but shall we remove backticks around package, module, etc. We use backticks only for keywords, references to names, etc.

Reply:
Addressed with 3e84be21c2dc33076f5c69ca3fe0b9bba41a2b89